fastlane/screenshots.json

```diff
@@ -21,243 +21,254 @@
 			"canvas_size": [1080,1920],
 			"text_size": [1080, 372],
 			"font_size": "84px",
-			"screenshot_size": [696, 1391],
-			"screenshot_offset": [193, 501],
+			"screenshot_size": [636, 1191],
+			"screenshot_offset": [226, 501],
 			"screenshot_mask": "playstoreres/assets/pixel-3-mask.png",
 			"device_frame": "playstoreres/assets/pixel-3.png",
-			"device_frame_size": [966, 1826],
-			"device_frame_offset": [58, 286]
+			"device_frame_size": [900, 1626],
+			"device_frame_offset": [91, 286]
 		},
 		{
 			"name": "Nexus 9",
 			"canvas_size": [3000,2000],
 			"text_size": [3000, 317],
 			"text_offset": [0, 0],
 			"font_size": "70px",
-			"screenshot_size": [2107, 1581],
-			"screenshot_offset": [446, 424],
+			"screenshot_size": [1731, 1298],
+			"screenshot_offset": [634, 493],
 			"device_frame": "playstoreres/assets/nexus-9.png",
-			"device_frame_size": [2715, 1827],
-			"device_frame_offset": [149, 301]
+			"device_frame_size": [2139, 1451],
+			"device_frame_offset": [430, 419]
 		}
 	],
 	"entries": [
+		// The world's most popular website builder
 		{
 			"device": "Pixel 3",
-			"filename": "images/phoneScreenshots/1-build-and-manage-your-website-pt1.png",
+			"filename": "images/phoneScreenshots/7-the-worlds-most-popular-website-builder.png",
 			"attachments": [
 				{
-					"file": "playstoreres/assets/attachments/Intro-1.png",
+					"file": "playstoreres/assets/attachments/phone-backgrounds/Circles-1.png",
 					"size": [1080, 1920],
 					"position": [0, 0]
 				},
 				{
-					"text": "playstoreres/metadata/%s/play_store_screenshot_7.html",
-					"size": [881,744],
-					"position": [110,75 ],
-					"font-size": 140,
-					"alignment": "left",
+					"file": "playstoreres/assets/attachments/themes/Themes-Android-Phone.png",
+					"size": [1080, 1438],
+					"position": [0, 465]
+				},
+				{
+					"text": "playstoreres/metadata/%s/play_store_screenshot_1.txt",
+					"size": [810,480],
+					"position": [127,0 ],
+					"alignment": "center",
 					"stylesheet": "playstoreres/assets/intro-text.css"
 				}
 			]
 		},
+		// Create a site or start a blog
 		{
 			"device": "Pixel 3",
-			"filename": "images/phoneScreenshots/1-build-and-manage-your-website-pt2.png",
+			"screenshot": "images/phoneScreenshots/1-create-a-site-or-start-a-blog.png",
+			"background": "playstoreres/assets/attachments/phone-backgrounds/Circles-2.png",
 			"attachments": [
 				{
-					"file": "playstoreres/assets/attachments/Intro-2.png",
-					"size": [1080, 1920],
-					"position": [0, 0]
+					"text": "playstoreres/metadata/%s/play_store_screenshot_2.txt",
+					"size": [750,350],
+					"position": [157,0 ],
+					"alignment": "center",
+					"stylesheet": "playstoreres/assets/intro-text.css"
```

Comment:
Rather than repeat a text attachment like this for all the screenshots (since most of the values are the same each time), you can set the text styles in the global stylesheet `playstoreres/assets/style.css` (defined in line 5) and set the `text_size` and `text_offset` for the device (e.g. line 22 for the Pixel 3) rather than set a `size` and `position` for each screenshot.

Then, you can set the `text` value for each screenshot like this:

```
		// Create a site or start a blog
		{
			"device": "Pixel 3",
			"screenshot": "images/phoneScreenshots/1-create-a-site-or-start-a-blog.png",
			"background": "playstoreres/assets/attachments/phone-backgrounds/Circles-2.png",
			"text": "playstoreres/metadata/%s/play_store_screenshot_2.txt"
		},
```

The exception to that is the first screenshot, since the text has to be added as an attachment there to overlay the other file attachments.